configure

```diff
@@ -2684,6 +2688,19 @@ if test "$xnvme" != "no" ; then
 fi
 print_config "xnvme engine" "$xnvme"
 
+##########################################
```

Comment:
I'd feel better if this actually tried to compile something using crc16_t10dif(crc, buffer, len) just so we know it works.

Reply:
Thanks for reviewing quickly, please have a look now.